wiki/Installation.md

```diff
@@ -2,7 +2,9 @@
 
 If you want more information about installing Swift packages with Xcode, [follow these instructions](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app). If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/).
 
-If you haven't added any Swift Packages to your project yet and thus don't have a `Package.swift` file, follow [these](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) instructions for adding your first package.
+To use Swift Package Manager in Xcode, click `File` -> `Swift Packages` -> `Add Package Dependency` and enter the Workflow repo's URL.
```

Comment:
I'm interested in the need of these lines in comparison to line 3 and what line 3 failed to provide that these lines are providing.

Reply:
For me, I felt that explicitly stating how to get the dependency in is worthwhile even if redundant alongside line 3. I could see still having this sentence from line 3:

"If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/)."

But I feel explicitly giving direction here is more easily digestible than linking out as the first sentence in line 3 does. As I'm writing this out, I'm thinking I probably could've combined those to be:

"To use Swift Package Manager in Xcode, click `File` -> `Swift Packages` -> `Add Package Dependency` and enter the Workflow repo's URL. If you want to learn more about Swift Package Manager, [this page goes into detail](https://swift.org/package-manager/)."

Reply:
I'll caution against instructions that rely on the UI. Are you going to update this if Xcode 14 changes the structure? Will you then have to have Xcode < 14 instructions and Xcode 14 instructions? Easier to link out to an Apple thing methinks.

Reply:
Updated to remove the new lines I had previously added, with a small change to the existing verbiage of line 3.